Makefile

```diff
@@ -493,8 +496,14 @@ images: #toil-docker
 .PHONY: init-conda
 init-conda:
 	echo "    started init-conda" >> $(LOGFILE)
-	$(CONDA) create -y -n $(VENV_NAME) python=$(VENV_PYTHON) pip=$(VENV_PIP)
-	$(CONDA) activate $(VENV_NAME) && pip install -U -r $(DIR)/etc/venv/requirements.txts
+	# source both bashrc and conda's script to be safe, so the conda command is found
+	source $(DIR)/bin/miniconda3/etc/profile.d/conda.sh && source /home/vagrant/.bashrc \
+		&& conda create -y -n $(VENV_NAME) python=$(VENV_PYTHON) pip=$(VENV_PIP)
```

Comment:
should still use $(CONDA) in order to ensure specified conda bin in $(DIR)  is being called. There are weird edge cases where  external conda bins could exist in PATH ahead of the newly sourced conda